fvm/errors/base.go

```diff
@@ -0,0 +1,218 @@
+package errors
+
+import (
+	"fmt"
+
+	"github.com/onflow/cadence/runtime"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// InvalidAddressError indicates that a transaction references an invalid flow Address
+// in either the Authorizers or Payer field.
+type InvalidAddressError struct {
+	address flow.Address
```

Comment:
We should let this field be accessible (possibly though getter) otherwise no point in storing it here

Reply:
fixed here 1cd7461